### PR TITLE
Feature/handle click on map

### PR DESCRIPTION
**Pull request by aistrate**

### CODE REVIEW

File: src/coffeeShops/CoffeeShopsChart.tsx [hunk 1]
```diff
@@ -49,28 +84,68 @@ function createOptions() {
...
+    },
   };
 }
 
 function tooltipTitle(tooltipItems: TooltipItem<"scatter">[]) {
-  return tooltipItems.map((item) => (item.raw as CoffeeShop).name).join("\n");
+  return tooltipItems.map((item) => (item.raw as DataPoint).title).join("\n");
+}
+
+function toChartCoordinates(
```
Comment (adrianbunea) [inline src/coffeeShops/CoffeeShopsChart.tsx:103]: https://www.chartjs.org/docs/latest/configuration/interactions.html#converting-events-to-data-values
This should help

**Review by adrianbunea — CHANGES_REQUESTED** [review]

File: src/coffeeShops/CoffeeShopsChart.tsx (hunk 1 above)
Comment (adrianbunea) [reply src/coffeeShops/CoffeeShopsChart.tsx:103]: I am referring to the helpers to convert click coordinates into Chart Points. `Chart.helpers.getRelativePosition(e, chart);`

Comment (aistrate) [reply src/coffeeShops/CoffeeShopsChart.tsx:103]: OK, I will change it.

Comment (aistrate) [reply src/coffeeShops/CoffeeShopsChart.tsx:103]: Made the change on the dev branch.

**Pull request merged**